sdk/servicebus/azure-servicebus/swagger/servicebus-swagger.json

```diff
@@ -310,9 +347,16 @@
           }
         },
         "AutoDeleteOnIdle": {
-          "description": "ISO 8061 timeSpan idle interval after which the queue is automatically deleted. The minimum duration is 5 minutes.",
+          "description": "ISO 8601 timeSpan idle interval after which the queue is automatically deleted. The minimum duration is 5 minutes.",
+          "type": "string",
+          "format": "iso8601",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "CreatedAt": {
```

Comment:
C# doesn't have"CreatedAt" in QueueProperties. It's in QueueMetrics. How about we also removed this and add back later if we decide to have CreatedAt in QueueProperties?

Reply:
swaggers are a representation of what the service responds with. In its response, it populates these fields. if these fields aren't exposed to users, I'd rather have a convenience method over it.